src/redux/greetingSlice.js

```diff
@@ -0,0 +1,27 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+const FETCH_GREETING = 'HELLO_WORLD/FETCH_DATA';
+
+export const fetchGreeting = createAsyncThunk(
+  FETCH_GREETING,
+  async () => {
+    const response = await fetch('http://127.0.0.1:1800/api/v1/greetings');
+    const data = await response.json();
+    return data;
```

Comment:
- [ ] Great work implementing the reqyuired changes from the previous reviewer. However, there's little issue here. Your default port value in the puma file from your backend is 3000, and when your app is opened, that should be what you should have on your address bar. (`http://127.0.0.1:3000/`). But when you wanted to fetch the api endpoint from frontend, you called this endpoint :point_right:  `const response = await fetch('http://127.0.0.1:1800/api/v1/greetings')` instead of this :point_right:  `const response = await fetch('http://127.0.0.1:3000/api/v1/greetings');`.
So with the endpoint you used, nothing shows up when I was clicking the `Click me` button until I changed the url to the later endpoint. Kindly look into this and fix.

### App behavious with your endpoint.
[your-endpoint.webm](https://user-images.githubusercontent.com/26597432/236636949-ca2f1ae1-f5c2-4f0e-b977-bf9e8099354a.webm)

### App behavios with my endpoint.
[my-endpoint.webm](https://user-images.githubusercontent.com/26597432/236636954-228b5066-0bb3-4a04-9ebb-329cc70cd656.webm)